modules/S05_firmware_details.sh

```diff
@@ -26,6 +26,10 @@ S05_firmware_details()
 
   print_output "[*] ""$(find "$FIRMWARE_PATH" "${EXCL_FIND[@]}" -type f | wc -l )"" files and ""$(find "$FIRMWARE_PATH" "${EXCL_FIND[@]}" -type d | wc -l)"" directories detected."
 
+  if [[ "$(find "$FIRMWARE_PATH" "${EXCL_FIND[@]}" -type f | wc -l)" -gt 0 ]] || [[ "$(find "$FIRMWARE_PATH" "${EXCL_FIND[@]}" -type d | wc -l)" -gt 0 ]];then
```

Comment:
we are doing the same search twice. Lets do it once, write the results into a variable and use this for your if statement